tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/message/payments/PaidMediaToTelegramPaidMedia.kt

```diff
@@ -0,0 +1,25 @@
+package dev.inmo.tgbotapi.types.message.payments
+
+import dev.inmo.tgbotapi.types.media.TelegramMediaPhoto
+import dev.inmo.tgbotapi.types.media.TelegramPaidMedia
+import dev.inmo.tgbotapi.types.media.TelegramPaidMediaPhoto
+import dev.inmo.tgbotapi.types.media.TelegramPaidMediaVideo
+
+fun PaidMedia.toTelegramPaidMedia(): TelegramPaidMedia? = when (this) {
+    is PaidMedia.Photo -> TelegramPaidMediaPhoto(
+        file = this.photo.biggest.fileId
+    )
+    is PaidMedia.Video -> TelegramPaidMediaPhoto(
+        file = this.video.fileId
+    )
+    is PaidMedia.Preview, is PaidMedia.Unknown -> null
+}
+
+fun PaidMedia.Video.toTelegramPaidMediaVideo(): TelegramPaidMediaVideo = TelegramPaidMediaVideo(
+        file = this.video.fileId,
+        duration = this.video.duration
+    )
```

Comment:
if I correctly remember, you may pass `width`/`height` to `TelegramPaidMediaVideo` too

Reply:
no, just InputFile

Reply:
Please, copy width, height and other possible parameters